tests/tensor/test_nlinalg.py

```diff
@@ -448,33 +470,57 @@ def test_wrong_type_of_ord_for_matrix(self):
 
     def test_non_tensorial_input(self):
         with pytest.raises(ValueError):
```

Comment:
Match a specific error message?
```suggestion
        with pytest.raises(ValueError, match="..."):
```